proto/p2p/p2p.proto

```diff
@@ -61,297 +61,357 @@ message Message {
+  // Consensus type to handle this message
   EngineType engine_type = 5;
 }
 
-// Message that contains the container ID and its bytes in response to "get".
-//
-// On receiving "put", the engine parses the container and tries to issue it to consensus.
+// Put is sent in response to Get with the requested block.
 message Put {
+  // Chain being responded from
   bytes chain_id = 1;
+  // Request id of the original Get request
   uint32 request_id = 2;
+  // Requested container
   bytes container = 3;
+  // Consensus type to handle this message
   EngineType engine_type = 4;
 }
 
-// Message that contains a preferred container ID and its container bytes
-// in order to query other peers for their preferences of the container.
-// For example, when a new container is issued, the engine sends out
-// "push_query" and "pull_query" queries to ask other peers their preferences.
-// See "avalanchego/snow/engine/common#SendMixedQuery".
+// PushQuery requests the preferences of a remote peer given a container.
 //
-// On receiving the "push_query", the engine parses the incoming container
-// and tries to issue the container and all of its parents to the consensus,
-// and calls "pull_query" handler to send "chits" for voting.
+// Remote peers should respond to a PushQuery with a Chits message
 message PushQuery {
+  // Chain being requested from
   bytes chain_id = 1;
+  // Unique identifier for this request
   uint32 request_id = 2;
+  // Timeout (ns) for this request
   uint64 deadline = 3;
+  // Container being gossiped
   bytes container = 4;
+  // Consensus type to handle this message
   EngineType engine_type = 5;
+  // Requesting peer's last accepted height
   uint64 requested_height = 6;
 }
 
-// Message that contains a preferred container ID to query other peers
-// for their preferences of the container.
-// For example, when a new container is issued, the engine sends out
-// "push_query" and "pull_query" queries to ask other peers their preferences.
-// See "avalanchego/snow/engine/common#SendMixedQuery".
+// PullQuery requests the preferences of a remote peer given a container id.
+//
+// Remote peers should respond to a PullQuery with a Chits message
 message PullQuery {
+  // Chain being requested from
   bytes chain_id = 1;
+  // Unique identifier for this request
   uint32 request_id = 2;
+  // Timeout (ns) for this request
   uint64 deadline = 3;
+  // Container id being gossiped
   bytes container_id = 4;
+  // Consensus type to handle this message
   EngineType engine_type = 5;
+  // Requesting peer's last accepted height
   uint64 requested_height = 6;
 }
 
-// Message that contains the votes/preferences of the node. It is sent in
-// response to a "push_query" or "pull_query" request.
-//
-// Upon receiving "chits", the engine will attempt to issue the preferred block
-// into consensus. If the referenced block is not locally available, the engine
-// will respond with a "get" message to fetch the missing block from the remote
```

Comment:
I generally got rid of any lower-level implementation details of consensus since I don't think that's something that should be a part of p2p's docs.